packages/flutter/lib/src/cupertino/checkbox.dart

```diff
@@ -165,6 +258,21 @@ class CupertinoCheckbox extends StatefulWidget {
   /// [RoundedRectangleBorder] with a circular corner radius of 4.0.
   final OutlinedBorder? shape;
 
+  /// True if this checkbox wants to show an error state.
```

Comment:
The word "wants to" is pretty weird. Maybe
```suggestion
  /// True if this checkbox is currently in an error state.
```